lib/modules/manager/gitlabci-include/extract.spec.ts

```diff
@@ -68,5 +69,37 @@ describe('modules/manager/gitlabci-include/extract', () => {
         expect(res?.deps[0].registryUrls).toEqual(['http://gitlab.test']);
       }
     });
+
+    it('supports multi-document files', () => {
+      const multiDocFile = codeBlock`
+        other:
+          content: to be ignored
+        ---
+        include:
+          - project: mikebryant/include-source-example
+            ref: 1.0.0
+        ---
+        include:
+          - project: mikebryant/include-source-example2
+            ref: 2.0.0
+        ---
+        more:
+          content: to be ignored
+      `;
+      const res = extractPackageFile(multiDocFile);
+      expect(res?.deps).toHaveLength(2);
+      expect(res?.deps).toMatchObject([
```

Comment:
```suggestion
      expect(res).toMatchObject([
```
prefer to match full results